index.bs

```diff
@@ -8686,6 +8906,18 @@ that correspond to the union's [=member types=].
         1.  If |types| includes {{object}}, then return the IDL value
             that is a reference to the object |V|.
     1.  If |V| [=is an Object=], then:
+        1.  If |types| includes an [=async iterable type=], then
+            1. If |types| does not include a [=string type=] or |V| does not have a \[[StringData]] [=/internal slot=], then
```

Comment:
Can we just say `, then return the result of converting V to an async iterable`, linking to [your conversion steps](https://whatpr.org/webidl/1397.html#ref-for-dfn-convert-ecmascript-to-idl-value%E2%91%A2%E2%91%A1) that seem to do the same thing as the below substeps?

Reply:
Those steps throw when no iterable method is present. Here we want to not throw in that case

Reply:
Oh duh 😅, thanks.